packages/larva-patterns/modules/carousel-slider/carousel-slider.prototype.js

```diff
@@ -12,6 +12,10 @@ const o_card = clonedeep( o_card_prototype );
 const o_indicator_prototype = require( '../../objects/o-indicator/o-indicator.prototype' );
 const o_indicator = clonedeep( o_indicator_prototype );
 
+c_heading.c_heading_text = 'Gallery Heading';
```

Comment:
This is a good idea so that we can use the same module for the one on Influencers, but the designs for the `carousel-slider` don't necessitate this heading text since we don't have data input for it in the block, unless we use the title of the curation module. We can ask Rosalie if she thinks this will be desired by some brands (I could definitely see that) and then come back to this PR!